### ISSUE TITLE
Loading from github

### ISSUE THREAD

**Issue by tischi** [mobie/mobie.github.io#12]
I am wondering whether this could, for convenience, also somehow be integrated into the FileAndUrlUtils:

```
private String adaptUrl( String url, String projectBranch )
	{
		if ( url.contains( "github.com" ) )
		{
			url = url.replace( "github.com", "raw.githubusercontent.com" );
			url += "/" + projectBranch;
		}
		return url;
	}
```

Not sure exactly how though.

**Comment by constantinpape** [comment]
I think this should go to gitbub utils: https://github.com/tischi/imagej-utils/blob/master/src/main/java/de/embl/cba/tables/github/GitHubUtils.java

**Comment by tischi** [comment]
Yes, partly, for sure, but `if ( url.contains( "github.com" ) )` does not make so much sense there I think, because that should be a given. I wonder more whether in the MoBIE code we need to _explicitly_ discover that the project location is github vs. somewhere else; or whether this could be _transparent_ and just handled by, e.g., FileAndUrlUtils.

**Comment by tischi** [comment]
Exactly something like this!
